### ISSUE TITLE
very veeery slow

### ISSUE THREAD

**Issue by vw117** [felixrieseberg/macintosh.js#93]
its loading at a snail's pace for me

**Comment by GrantMeStrength** [comment]
Yes, I'm trying it on a Raspberry Pi 4 and it's still loaded the dependencies after ten minutes.

**Comment by Itai-Nelken** [comment]
yes, its really slow on a pi, I gave up.
QEMU is the best way for getting old versions of MacOS running on a pi.
for example, take a look [here](https://github.com/Itai-Nelken/RPi-QEMU-VM-scripts).

**Comment by GrantMeStrength** [comment]
I used the diagnostics mode, and oops - lots of things are going wrong as it starts up.
